### PR TITLE
(NFC) Fix indenting of Case DashBoard.tpl

### PR DESCRIPTION
**Pull request by seamuslee001**
Overview
----------------------------------------
This just fixes the Indenting of lines within the Dashboard.tpl file. No Functional Changes included. 

Before
----------------------------------------
Indenting all over the place

After
----------------------------------------
Indenting standardised

ping @eileenmcnaughton

### CODE REVIEW

**Comment by eileenmcnaughton** [comment]
@seamuslee001 so in addition to indenting changes you move a div 

https://github.com/civicrm/civicrm-core/pull/12849/files?w=1

That seems best in a separate PR?

**Comment by seamuslee001** [comment]
@eileenmcnaughton I don't think so if you look at https://github.com/civicrm/civicrm-core/blob/682a72f986f02bcdc4ad38d53e8b2e55406d1cfb/templates/CRM/Case/Page/DashBoard.tpl#L29 you will notice that there is a div outside of the {if} here, https://github.com/civicrm/civicrm-core/pull/12848/files they removed the div that was on the outside of the if rather than the one on the inside of the if (they were doubling each other up causing issues on template). The correct way seems to be at the bottom if (indented by 2 spaces) then a div (no indent)

File: templates/CRM/Case/Page/DashBoard.tpl [hunk 1]
```diff
@@ -49,65 +49,65 @@
...
-<div class="spacer"></div>
-    <h3>{if $myCases}{ts}My Cases With Recently Performed Activities{/ts}{else}{ts}All Cases With Recently Performed Activities{/ts}{/if}</h3>
-    {if $recentCases}
-    {include file="CRM/Case/Form/CaseFilter.tpl" context="$context" list="recent"}
-    <div class="form-item">
-        {include file="CRM/Case/Page/DashboardSelector.tpl" context="dashboard" list="recent" all="$all"}
-    </div>
-    {else}
+    <div class="spacer"></div>
+      <h3>{if $myCases}{ts}My Cases With Recently Performed Activities{/ts}{else}{ts}All Cases With Recently Performed Activities{/ts}{/if}</h3>
```
Comment (eileenmcnaughton) [inline templates/CRM/Case/Page/DashBoard.tpl]: So here - the <h3> is not inside the div & so it should be directly below (not indented)

**Comment by eileenmcnaughton** [comment]
I noted one area where the indent is not quite right but fixing that seems optional - this does make the layout more correct & I could not see any adverse effects. Merging

**Pull request merged**